test/db/cmd/cmd_idp

```diff
@@ -10,6 +10,7 @@ EXPECT=<<EOF
 0x00401000 0 pdb.void___cdecl_SomeCoolFunction_void
 0x00401010 0 pdb._main
 EOF
+BROKEN=1
```

Comment:
Any particular reason why this is broken? It is not a class or a bitfield, should be supported by the current types model

Reply:
Ah, I forgot it..

Reply:
This is still marked as BROKEN